src/Microsoft.ML.Onnx/ProtabilityCatalog.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Data;
+using Microsoft.ML.Model.Onnx;
+using Microsoft.ML.UniversalModelFormat.Onnx;
+
+namespace Microsoft.ML
+{
+    public static class ProtabilityCatalog
```

Comment:
>ProtabilityCatalog [](start = 24, length = 18)

Should probably be named `OnnxExportExtensions` or somesuch like this.

Reply:
No problem. File name will be changed accordingly.

---
In reply to: [245427449](https://github.com/dotnet/machinelearning/pull/2013#discussion_r245427449) [](ancestors = 245427449)